backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestEventRepository.java

```diff
@@ -33,6 +33,14 @@
 
   public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
 
+  @Query(
+      value =
+          " SELECT DISTINCT ON (patient_id) *, COALESCE(date_tested_backdate, created_at) AS test_date FROM {h-schema}test_event"
```

Comment:
I might change the `test_date` alias to something more obviously query-local—since we do this same coalesce in the Java code later if somebody calls `getDateTested` and we wouldn't want somebody to look at this and think that this information was something that we could usefully manipulate in the query.

Reply:
Done in 14ee23a7